app/controllers/questions_controller.rb

```diff
@@ -110,9 +110,8 @@ def unanswered
     @title = 'Unanswered questions'
     @questions = Node.questions
       .where(status: 1)
-      .includes(:answers)
-      .references(:answers)
-      .where(answers: { id: nil })
+      .left_outer_joins(:comments)
```

Comment:
Great choice using the outer joins here :1st_place_medal:

Reply:
Thanks!